### PR TITLE
Add links to design checklist

### PR DESCRIPTION
**Pull request by mperrotti**

### CODE REVIEW

File: content/guides/contribute/design.mdx [hunk 1]
```diff
@@ -41,14 +39,13 @@ There are different ways you can ask for and get feedback:
...
-  
+All design patterns for Primer Library consideration should at minimum:
+
+- [ ] **Leverage Primer**: Leverages the foundational building blocks and tools of Primer. Reuses or extends existing patterns and primitives rather than creating overrides
+  - [ ] Colors, spacing, sizes, and typographic styles from [@primer/primitives](https://github.com/primer/primitives/)
+  - [ ] All icons come from [@primer/octicons](https://github.com/primer/octicons/)
+  - [ ] Considers the criteria in the [responsive design guidelines](https://primer.style/design/foundations/responsive)
+- [ ] **Maintain high design quality:** Visually in harmony with GitHub UI and all visual elements are used are in their correct context
+- [ ] **Prioritize code as a top concern:** Designed with code quality and component API top of mind
+- [ ] **Is inclusive:** Designs follow Primer [accessibility guidelines](https://primer.style/design/guides/accessibility/guidelines) and considers a global audience
```
Comment (emilybrick) [inline content/guides/contribute/design.mdx:50]:
```suggestion
- [ ] **Be inclusive:** Designs follow Primer [accessibility guidelines](https://primer.style/design/guides/accessibility/guidelines) and considers a global audience
```

ugh. I know this is an aside from the original PR intent - I think I wrote this (or the bullet below) but grammatically this feels weird - should maybe be `all design patterns at a minimum should **be inclusive**' (?) the rest of the bullets sound write as a continuation of the top point

File: content/guides/contribute/design.mdx [hunk 2]
```diff
@@ -23,11 +23,9 @@ When creating a new pattern, we advise starting with an audit of similar pattern
 
 Finally, consider where a pattern will live in the context of a system. A pattern designed for a specific context is not likely to be considered a reusable component.
 
-
-
 ## Share early, and often
 
-There are many opportunities to get feedback from Primer maintaners. 
+There are many opportunities to get feedback from Primer maintaners.
```
Comment (raytalks) [inline content/guides/contribute/design.mdx:28]: typo: maintainers

**Review by raytalks — COMMENTED** [review]
Just one tiny typo ☺️

**Review by tallys — APPROVED** [review]
Maintaners

**Pull request merged**